conans/client/recorder/search_recorder.py

```diff
@@ -5,16 +5,27 @@ class _SearchRecipe(namedtuple("SearchRecipe", "ref")):
     with_packages = True
 
     def to_dict(self):
-        data = {"id": repr(self.ref)}
+        try:
+            recipe_name = repr(self.ref).split('/')[0]
+        except Exception:
+            recipe_name = repr(self.ref)
+
+        data = {
+            "name": recipe_name,
+            "id": repr(self.ref)
+        }
         return data
 
 
 class _SearchPackage(namedtuple("SearchPackage",
                                 "package_id, options, settings, requires")):
-
     def to_dict(self):
-        return {"id": self.package_id, "options": self.options, "settings": self.settings,
-                "requires": self.requires}
+        return {
+            "id": self.package_id,
```

Comment:
Things like ``package_id`` will no longer make sense in the new search. In any case the ``SearchRecorder`` will probably dissapear or get a big refactor.